### PR TITLE
feat: update version

### PR DESCRIPTION
**Pull request by yacheng**
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).
5. Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.

### CODE REVIEW

File: packages/driver-universal/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "driver-dom": "^2.0.5",
     "driver-kraken": "^0.1.0",
     "driver-miniapp": "^0.1.0",
-    "driver-weex": "^1.0.0",
+    "driver-weex": "^2.0.0",
```
Comment (yuanyan) [inline packages/driver-universal/package.json]: 如果 weex 是 breakchange 升级，需要确保 driver-universal 是不是 breakchange 升级？

Comment (yacheng) [reply packages/driver-universal/package.json]: driver-universal 上一次大版本升级 weex 链路一直没跑起来，这个 pr 应该算作 bugfix，修复上一次 driver-universal 的 breakchange

File: packages/driver-universal/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "driver-universal",
-  "version": "3.1.0",
+  "version": "3.1.1",
```
Comment (wssgcg1213) [inline packages/driver-universal/package.json]: 3.1.1 已经发了, 需要换一个版本号

**Comment by wssgcg1213** [comment]
LGTM

**Comment by wssgcg1213** [comment]
冲突看下

**Comment by yacheng** [comment]
已合并 master

**Comment by wssgcg1213** [comment]
+1

**Pull request merged**